README.md

```diff
@@ -1,52 +1,162 @@
-# Scotch
+# EasyVCR
 
-### Disclaimer
-This is a fork of the original [Scotch by Martin Leech](https://github.com/mleech/scotch), updated to address outstanding functionality issues, as well as introduce a handful of new features. This version of Scotch is a C# rewrite of the original F# version.
+#### Disclaimer
+This is a distant fork of the original [Scotch by Martin Leech](https://github.com/mleech/scotch), completely rewritten in C# and enhanced with several new features and performance improvements.
 
-## What is Scotch?
-Scotch is a library for recording and replaying HTTP interactions in your test suite. 
+## What is EasyVCR?
+EasyVCR is a library for recording and replaying HTTP interactions in your test suite.
 
 This can be useful for speeding up your test suite, or for running your tests on a CI server which doesn't have connectivity to the HTTP endpoints you need to interact with.
 
-Scotch is based on the [VCR Ruby gem](https://github.com/vcr/vcr).
-
-### Step 1.
+## How to use EasyVCR
+#### Step 1.
 Run your test suite locally against a real HTTP endpoint in recording mode
 
 ```csharp
-using EasyPost.Scotch;
+using EasyPost.EasyVCR;
+
+// Create a cassette to handle HTTP interactions
+var cassette = new Cassette("path/to/cassettes", "my_cassette");
 
-// Create a HttpClient which uses a RecordingHandler
-var vcrHttpClient = HttpClients.NewHttpClient(pathToCassetteFile, ScotchMode.Recording);
+// create an HttpClient using the cassette
+var recordingHttpClient = HttpClients.NewHttpClient(cassette, Mode.Record);
 
 // Use this HttpClient in any class making HTTP calls
 // For example, RestSharp v107+ supports custom HTTP clients
-RestClient restClient = new RestClient(vcrHttpClient, new RestClientOptions()));
+RestClient restClient = new RestClient(recordingHttpClient, new RestClientOptions()));
+
+// Or make HTTP calls directly
+var response = await recordingHttpClient.GetAsync("https://api.example.com/v1/users");
 ```
 Real HTTP calls will be made and recorded to the cassette file.
 
-### Step 2.
+#### Step 2.
 Switch to replay mode:
 ```csharp
```

Comment:
There should be line breaks between all these items. Double check the whole README for more of these too.